### PR TITLE
Feature/fix remarks from review

### PR DESCRIPTION
**Pull request by wo0dyn**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -42,3 +42,7 @@ coverage.xml
 
 # Sphinx documentation
 docs/_build/
+
+# OSX
+.DS_Store
```
Comment (twidi) [inline .gitignore:47]: https://twitter.com/jy/status/479252400742871040 and https://twitter.com/Horse_iOS/status/478690729678479360 :)

File: oscar_impersonate/templates/oscar_impersonate/partials/buttons.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{% comment %}
+
+Variables
+    - request.user:             staff (who's access to Oscar dashboard)
+    - customer:                 current customer
+    - request.impersonator:     staff that impersonates a customer
+    - btn_start_css_class:      CSS classes for impersonation start button
+    - btn_stop_css_class:       CSS classes for impersonation stop button
+
+{% endcomment %}
```
Comment (twidi) [inline oscar_impersonate/templates/oscar_impersonate/partials/buttons.html:12]: LOVE THAT ! One more thing i'll check now for all reviews, thanks a lot :p

File: .gitignore (hunk 1 above)
Comment (wo0dyn) [inline .gitignore:47]: Grrr… Actually, that thing should have been put in my `~/.gitignore_global`. :wink:

File: oscar_impersonate/templates/oscar_impersonate/partials/buttons.html (hunk 2 above)
Comment (wo0dyn) [inline oscar_impersonate/templates/oscar_impersonate/partials/buttons.html:12]: Glad you like it :blush:

**Comment by twidi** [comment]
Did you see

https://github.com/oscaro/django-oscar-impersonate/commit/2e0b6fa528be156a40877f2cbdb0a59912ee5cd0#diff-88b99bb28683bd5b7e3a204826ead112R67

and

https://github.com/oscaro/django-oscar-impersonate/pull/2#discussion_r13914060

?

**Comment by twidi** [comment]
ok for both, good to merge

**Comment by wo0dyn** [comment]
Thx!

**Pull request merged**